Adding docs for userroles and permissions on user management dashboard. (#752)
supertokens/docs: v2/emailpassword/user-roles/managing-roles-and-permissions.mdx

- Get a list of all permissions assigned to a role
- Get a list of all roles that have a specific permission

:::tip

> You can easily add, remove and edit permissions for a role from the user management dashboard. To know more about how to manage your user roles and permissions from user management dashboard look <a href="/docs/userdashboard/managing-user-roles-and-permissions" target="_blank">here</a>

Reviewer: ```suggestion
You can also add, remove and edit permissions for a role from the user management dashboard. To know more about how to manage your user roles and permissions from user management dashboard see <a href="/docs/userdashboard/managing-user-roles-and-permissions" target="_blank">this page</a>.
```
